app/javascript/dashboard/routes/dashboard/settings/profile/Index.vue

```diff
@@ -275,6 +275,71 @@ export default {
     >
       <ChangePassword />
     </FormSection>
+    <FormSection
+      :title="$t('PROFILE_SETTINGS.FORM.SECURITY_SECTION.TITLE')"
+      :description="$t('PROFILE_SETTINGS.FORM.SECURITY_SECTION.NOTE')"
+    >
+      <div class="flex flex-col space-y-4">
+        <div
+          class="bg-slate-50 dark:bg-slate-900 rounded-lg p-4 border border-slate-200 dark:border-slate-700"
```

Comment:
@tds-1 The colors used are slate-50 slate-200. These are old colors in the application. We will get rid of them soon. 

Can you use the next colors? Those have `n-` prefix. Please search the codebase for the same.